doc/en/api/1.0.0/styles.yaml

```diff
@@ -643,19 +643,121 @@ parameters:
     description: >
 
       The style body of a request
-      JSON or XML style uploads follow this schema. For example:
+      JSON or XML style uploads follow the schema. For example:
 
         - application/xml: 
           
           ```
-          <style><name>roads_style</name><filename>roads.sld</filename></style>
+          <style>
+            <name>roads_style</name>
+            <filename>roads.sld</filename>
+          </style>
           ```
 
         - application/json: 
         
-            ```
-            {"style": { "name": "roads_style", "filename": "roads.sld"}}
-            ```
+          ```
+          {
+            "style": { 
+              "name": "roads_style", 
+              "filename": "roads.sld"
+            }
+          }
+          ```
+
+      Otherwise, the style body is an actual style:
+
+        - application/zip: 
+        
+          ```
+          <ZIP file containing SLD and icons>
```

Comment:
Is it worth listing the directory structure of the zip as an example?

```
point_of_interest.sld
landmark.png
park.png
OpenSans.ttf
...
```

Icons can also be managed with the resource endpoint.